crypto/x509/x509_local.h

```diff
@@ -9,6 +9,16 @@
 
 #include "internal/refcount.h"
 
+# define KU_DIGITAL_SIGNATURE    0x0080
```

Comment:
No, this is a wrong fix. Instead the https://github.com/openssl/openssl/blob/master/include/openssl/x509.h.in#L361 should include openssl/http.h and OCSP_REQ_CTX_nbio_d2i should be replaced with OSSL_HTTP_REQ_CTX_nbio_d2i. (Look at how OCSP_REQ_CTX_nbio_d2i is defined.)

If we avoid including ocsp.h from x509.h there should be no clash when including x509.h.